src/postprocess/chi0.jl

```diff
@@ -245,9 +317,16 @@ Get the density variation δρ corresponding to a total potential variation δV.
 Note: This function assumes that all bands contained in `ψ` and `eigenvalues` are
 sufficiently converged. By default the `self_consistent_field` routine of `DFTK`
 returns `3` extra bands, which are not converged by the eigensolver
-(see `n_ep_extra` parameter). These should be discarded before using this function.
+(see `n_ep_extra` parameter). These should be discarded from `ψ` before using this
+function. However, one can still use additional information from nonconverged bands
+stored in `ψ_extra` (typically, the `3` extra bands returned by default in SCF
+routine).
 """
-function apply_χ0(ham, ψ, εF, eigenvalues, δV; kwargs_sternheimer...)
+function apply_χ0(ham, ψ, εF, eigenvalues, δV;
+                  ψ_extra=[zeros(size(ψk,1), 0) for ψk in ψ],
+                  ε_extra=[zeros(0) for ψk in ψ],
```

Comment:
```suggestion
                  ψ_extra=[zeros(size(ψk, 1), 0) for ψk in ψ],
                  eigenvalues_extra=[zeros(0) for _ in eigenvalues],
```

Reply:
Isn't `eigenvalues_extra` too long ?

Reply:
yes, but it's consistent. You have tab completion and all.